src/pages/settings/user/components/Password.tsx

```diff
@@ -49,6 +54,7 @@ export function Password() {
           onChange={(event: ChangeEvent<HTMLInputElement>) =>
             setPassword(event.target.value)
           }
```

Comment:
Can we replace this with `onValueChange` ?

Reply:
Sure, I did not see that we can refactor this part. Changed!